csrc/fusion_segmenter.cpp

```diff
@@ -2633,15 +2633,18 @@ void deDuplicateScalarExprs(std::vector<Expr*>& exprs) {
 } // namespace
 
 std::optional<std::unique_ptr<HeuristicParams>> SegmentedGroup::
-    getMaybeHeuristicParams(SchedulerRuntimeInfo& runtime_info) {
+    getMaybeHeuristicParams(
+        SchedulerRuntimeInfo& runtime_info,
+        bool skip_compile_time_checks) {
```

Comment:
Isn't this always `true`? If so, can we remove the parameter?

Reply:
You're right. It's only called in `FusionKernelRuntime::getMaybeHeuristicsFor`. I'll remove the option.